src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/util/CaseListUtils.kt

```diff
@@ -0,0 +1,66 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util
+
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.ManagedCase
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**suggestion:** ‏As mentioned above, this is part of the Eligibility Service which could be used instead.